components/Navigation.tsx

```diff
@@ -13,6 +13,9 @@ const Navigation: React.FC = () => (
       <Link href="/about">
         <a>About</a>
       </Link>
+      <Link href="/faq">
```

Comment:
This is only a placeholder component. Please refrain from adding to this file until we replace it with a new navigation component.